pkg/proxy/healthcheck/service_health.go

```diff
@@ -174,13 +174,15 @@ func (h hcHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
 		`, h.name.Namespace, h.name.Name, count)), "\n"))
 }
 
-func (hcs *server) SyncEndpoints(newEndpoints map[types.NamespacedName]int) error {
+func (hcs *server) SyncEndpoints(newEndpoints map[types.NamespacedName]int, noHealthCheck map[types.NamespacedName]bool) error {
 	hcs.lock.Lock()
 	defer hcs.lock.Unlock()
 
 	for nsn, count := range newEndpoints {
 		if hcs.services[nsn] == nil {
-			klog.V(3).Infof("Not saving endpoints for unknown healthcheck %q", nsn.String())
+			if !noHealthCheck[nsn] {
+				klog.V(3).Infof("Not saving endpoints for unknown healthcheck %q", nsn.String())
+			}
```

Comment:
```suggestion
```

just continue and don't log, IIUIC if the service is not present in the struct is because it will not need healthcheck